### PR TITLE
Kubernetes command for copy binaries for services

### PR DESCRIPTION
**Pull request by knabben**
Encapsulation of runners for command reuse, simplifying interface with global flags parsing

### CODE REVIEW

File: experiments/swdt/cmd/kubernetes.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"swdt/apis/config/v1alpha1"
+	"swdt/pkg/pwsh/executor"
+	"swdt/pkg/pwsh/kubernetes"
```
Comment (mloskot) [inline experiments/swdt/cmd/kubernetes.go:23]: This package seems missing to me

Comment (knabben) [reply experiments/swdt/cmd/kubernetes.go:23]: lol what...

Comment (mloskot) [reply experiments/swdt/cmd/kubernetes.go:23]: Unless you prefer to work on it, shall I do it? I'd need some bearings on canonical K8s project CI or its just GHA workflow w/ Go linters and `go test` jobs?

Comment (knabben) [reply experiments/swdt/cmd/kubernetes.go:23]: feel free to add it, we have some other projects using like https://github.com/kubernetes-sigs/windows-operational-readiness/tree/main/.github/workflows

Comment (mloskot) [reply experiments/swdt/cmd/kubernetes.go:23]: It looks like you've already added it a minute ago in https://github.com/kubernetes-sigs/sig-windows-dev-tools/commit/a4b06be65f5a6e146318ec6d73229b5b3e1de267 - not the right approach?

Comment (mloskot) [reply experiments/swdt/cmd/kubernetes.go:23]: @knabben By the way, what is the K8s sentiment to use [pre-commit ](https://pre-commit.com/) hooks? Is it in any way popular as a first line of attack w.r.t Go linting and testing?

**Comment by friedrichwilken** [comment]
good to see you are working on this so actively!

File: experiments/swdt/cmd/kubernetes.go (hunk 1 above)
Comment (knabben) [reply experiments/swdt/cmd/kubernetes.go:23]: i do prefer to run this on CI, it can be fine if optional though.. have a linter configured on my IDE

Comment (mloskot) [reply experiments/swdt/cmd/kubernetes.go:23]: yeah, optional, definitely and as long as you don't run `pre-commit install`  in he SWDT repo, you don't suffer :)

Comment (knabben) [reply experiments/swdt/cmd/kubernetes.go:23]: can you open a PR for it?

**Comment by mloskot** [comment]
*approve* :)

File: experiments/swdt/cmd/kubernetes.go (hunk 1 above)
Comment (mloskot) [reply experiments/swdt/cmd/kubernetes.go:23]: yes, I am going to, not sure if tonight, but tomorrow for sure.

**Review by mloskot — APPROVED** [review]

**Pull request merged**